src/Form/BlogSearchForm.php

```diff
@@ -119,13 +146,12 @@ protected function getFormActions()
      */
     protected function getFormValidator()
     {
-        $validator = new RequiredFields(array(
-            'Email'
-        ));
+        $validator = RequiredFields::create([
+            'Email',
```

Comment:
I just noticed that `Email` is a required field. Was this from copying an example during the initial build? I can remove and update the PR if needed.